Misc/ACKS

```diff
@@ -402,6 +402,7 @@ Lisandro Dalcin
 Darren Dale
 Andrew Dalke
 Lars Damerow
+Hauke Dämpfling
 Evan Dandrea
```

Comment:
i don't know whether there's a specified collation, but Mäki and Müller appear after Murray in this file.